src/main/java/it/reply/orchestrator/service/CmdbServiceImpl.java

```diff
@@ -272,9 +272,14 @@ public CloudProvider fillCloudProviderInfo(String providerId,
                 .collect(Collectors.toList());
             List<Image> imageList = new ArrayList<>();
             List<Flavor> flavorList = new ArrayList<>();
-            for (Tenant tenant : tenantList) {
+            if (!tenantList.isEmpty()) {
+              //only one element must be here
```

Comment:
why?

Reply:
To avoid exception in the following Tenant tenant = tenantList.get(0);
For removed to avoid break (the devil) at the end of first cycle; i need only one (the first) entry

Reply:
ok, I see